### ISSUE TITLE
Travis failing due to SSL error

### ISSUE THREAD

**Issue by Ainali** [publiccodenet/standard#344]
Builds can fail due to SSL error ([example](https://travis-ci.com/github/publiccodenet/standard/builds/172972920)). How do we want to resolve that?

**Comment by bvhme** [comment]
Related, work on https://github.com/publiccodenet/about/issues/695 to reduce build errors for timeouts.

**Comment by bvhme** [comment]
The solution from publiccodenet/about#695 would not work here since the SSL checks are already off. I'd be interested to know what is wrong with this site more specifically.

**Comment by ericherman** [comment]
I would like some help understanding these build failures, as they do not seem to be SSL specifically, e.g.:

  *  External link http://www.bpmn.org/ failed: response code 0 means something's wrong.
  *  External link https://www.youtube.com/watch?v=_cyI7DMhgYc failed: 429 No error

I have never produced these errors locally. Is there more we can do to tune our CI config?

```
$ PAGES_REPO_NWO=publiccodenet/standard script/test.sh
Configuration file: /home/eric/src/publiccodenet-standard/_config.yml
            Source: /home/eric/src/publiccodenet-standard
       Destination: /home/eric/src/publiccodenet-standard/_site
 Incremental build: disabled. Enable with --incremental
      Generating... 
Invalid theme folder: _sass
      Remote Theme: Using theme publiccodenet/jekyll-theme
   GitHub Metadata: No GitHub API authentication could be found. Some fields may be missing or have incorrect data.
                    done in 2.08 seconds.
 Auto-regeneration: disabled. Use --watch to enable.
Running ["LinkCheck", "ImageCheck", "ScriptCheck"] on ["./_site"] on *.html... 


Checking 88 external links...
Ran on 29 files!


HTML-Proofer finished successfully.
```

**Comment by bvhme** [comment]
The error 0 is a server not responding with an error or the server never being reached at all. The error 429 is usually throttling.

The solution we employ elsewhere is to just ignore 0 and 429 errors, suboptimal, but until these others get their house in order it is what it is.

**Comment by Ainali** [comment]
Fixed by #380
